airflow/dag_processing/manager.py

```diff
@@ -1146,8 +1077,8 @@ def _collect_results_from_processor(self, processor, session: Session = NEW_SESS
 
         span.end(end_time=datetime_to_nano(last_finish_time))
 
-        Stats.timing(f"dag_processing.last_duration.{file_name}", last_duration)
-        Stats.timing("dag_processing.last_duration", last_duration, tags={"file_name": file_name})
+        Stats.timing(f"dag_processing.last_duration.{file_name}", last_duration * 1000.0)
+        Stats.timing("dag_processing.last_duration", last_duration * 1000.0, tags={"file_name": file_name})
```

Comment:
Does this change the unit for stats?

Oh welp no, we are defaulting to seconds -- converting to ms here

Reply:
Yeah, previously this was a timedelta object, now it's a plain float, so we need to convert it to ms, right?